README.md

```diff
@@ -15,6 +15,7 @@ const ws = new WriteStream(feed)
 const rs = new ReadStream(feed, {
   start: 0,
   live: true
```

Comment:
missing a , at the end here now if i'm being a bit pendantic

Reply:
fixed in https://github.com/mafintosh/hypercore-streams/pull/2/commits/6c921b7a21e61ae49e7d90c05a1ea720feeee824